beta-20160602.md

```diff
@@ -0,0 +1,29 @@
+---
+title: What's New in beta-20160602
+toc: false
+---
+
+## Jun 2, 2016
+
+### New Features
+
+* String literals can now be parsed as `DATE`, `TIMESTAMP`, `TIMESTAMPTZ` or `INTERVAL` without an explicit cast. [#6925](https://github.com/cockroachdb/cockroach/pull/6925)
```

Comment:
I wasn't going to do this, but since there's already a comment on this PR: this should use an oxford comma. L12 does.